administration/src/bp-modules/stores/StoresCSVInput.tsx

```diff
@@ -83,9 +87,10 @@ const StoresCsvInput = ({ setAcceptingStores, fields }: StoresCsvInputProps): Re
         return
       }
 
-
-      if (!lines.every((line: string[]) => line.length === numberOfColumns)) {
-        showInputError('Keine gültige CSV Datei. Nicht jede Reihe enthält gleich viele Elemente.')
+      if (lines.some((line: string[]) => line.length !== headers.length)) {
+        showInputError(
+          `Die CSV enthält eine ungültige Anzahl an Spalten. Es sind maximal ${headers.length} Spalten erlaubt.`
```

Comment:
_pretty much optional:_
if I upload less columns that required and see the error with "Es sind maximal ${headers.length} Spalten erlaubt", this might sound like I have an extra column rather than a missing one.
I'd say the first part of the error "Die CSV enthält eine ungültige Anzahl an Spalten" sounds accurate enough here, but maybe that's just my personal feeling.

Reply:
removed that